src/mount.rs

```diff
@@ -48,6 +48,59 @@ libc_bitflags!(
 
 libc_bitflags!(
     pub struct MntFlags: c_int {
+        /// Read only filesystem
+        MNT_RDONLY;
```

Comment:
You need to `#[cfg(...)]` gate each of these according to which OSes have it.

Reply:
Added macOS target.
Should I add more OS targets?

Reply:
Yes, it would be great to add other targets.  I think most of these flags are also supported by one or more of the BSDs, for example.  Also, please don't force push until I ask you to squash your commits.  It makes incremental review harder.

Reply:
OK, thanks for your reminder. I added more BSD-like targets.

Reply:
@asomers any comments?

Reply:
Just remove bitrig and add a CHANGELOG entry.